CorePayments/src/test/java/com/paypal/android/corepayments/RestClientUnitTest.kt

```diff
@@ -0,0 +1,163 @@
+package com.paypal.android.corepayments
+
+import io.mockk.CapturingSlot
+import io.mockk.coEvery
+import io.mockk.mockk
+import io.mockk.slot
+import junit.framework.TestCase.assertEquals
+import junit.framework.TestCase.assertNull
+import junit.framework.TestCase.assertSame
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import java.net.URL
+
+@ExperimentalCoroutinesApi
+@RunWith(RobolectricTestRunner::class)
+class RestClientUnitTest {
+
+    private val apiGETRequest = APIRequest("sample/path", HttpMethod.GET, null)
+
+    private val requestBody = """{ "sample": "json" }"""
+    private val apiPOSTRequest = APIRequest("sample/path", HttpMethod.POST, requestBody)
+
+    private val httpSuccessResponse = HttpResponse(200)
+
+    private val sandboxConfig = CoreConfig("sandbox-access-token", Environment.SANDBOX)
+    private val stagingConfig = CoreConfig("staging-access-token", Environment.STAGING)
+    private val liveConfig = CoreConfig("live-access-token", Environment.LIVE)
+
+    private lateinit var http: Http
+    private lateinit var httpRequestSlot: CapturingSlot<HttpRequest>
+
+    private lateinit var sut: RestClient
+
+    @Before
+    fun beforeEach() {
+        http = mockk(relaxed = true)
+        httpRequestSlot = slot()
+    }
+
+    @Test
+    fun `send() should properly format the url for the sandbox environment`() = runTest {
+        coEvery { http.send(capture(httpRequestSlot)) } returns httpSuccessResponse
+
+        sut = RestClient(sandboxConfig, http, "en_US")
+        sut.send(apiGETRequest)
+
+        val httpRequest = httpRequestSlot.captured
+        assertEquals(URL("https://api.sandbox.paypal.com/sample/path"), httpRequest.url)
+    }
+
+    @Test
+    fun `send() should properly format the url for the staging environment`() = runTest {
```

Comment:
We don't support staging on iOS. I think we should remove it from Android too. I don't think it's accessible to merchants.

Reply:
We should make a ticket for it 🏈